_drafts/contribute-to-ansible.markdown

```diff
@@ -183,39 +183,243 @@ $ source hacking/env-setup
 
 Install base dependencies:
 ```sh
-$ pip install -r requirements.txt
+$ pipenv install -r requirements.txt
 ```
 
 Ansible come with a lots of development dependencies, you can install all or just a subset
 in concordance with your development purpose.
 
 If you work on documentation all development dependencies are presents in `./docsite_requirements.txt`.
 ```sh
-$ pip install -r ./docsite_requirements.txt
+$ pipenv install -r ./docsite_requirements.txt
 ```
 
 To working on bugfix, tests, or new features you need to install specific requirements:
 ```sh
 $ # minimal subset of requirements to install
-$ pip install -r ./test/runner/requirements/units.txt
-$ pip install -r ./test/runner/requirements/integration.txt
-$ pip install -r ./test/runner/requirements/coverage.txt
-$ pip install -r ./test/runner/requirements/constraints.txt
-$ pip install -r ./test/runner/requirements/ansible-test.txt
-$ pip install -r ./test/runner/requirements/sanity.txt
+$ pipenv install -r ./test/runner/requirements/units.txt
+$ pipenv install -r ./test/runner/requirements/integration.txt
+$ pipenv install -r ./test/runner/requirements/coverage.txt
+$ pipenv install -r ./test/runner/requirements/constraints.txt
+$ pipenv install -r ./test/runner/requirements/ansible-test.txt
+$ pipenv install -r ./test/runner/requirements/sanity.txt
 ```
 
 #### c. Setup your docker environment
     c. Créer des hotes à configurer (docker)
 
 ## Step 2 - Take action!
-    a. find issue on ansible
-    b. apply coding best practices
-    http://docs.ansible.com/ansible/latest/community.html#for-current-and-prospective-developers
-    c. test
-    http://docs.ansible.com/ansible/latest/dev_guide/testing.html
-    d. Git : rebase, cherry-pick, squash, --amend
-    e. push on your github repository
+
+#### a. Find issues on ansible
+Now is time to take actions! Now you'll be able to see what's going on under the hood of Ansible.
+First you need find a subject to workon, well ansible come 
+from an issues tracker where you can find ideas or ways of working.
```

Comment:
First you need helps you to find a subject to work on. Ansible gets an issue tracker which will helps you to find ideas or ways of work.